website/content/api-docs/secret/aws.mdx

```diff
@@ -670,6 +670,10 @@ This endpoint creates or updates static role definitions. A static role is a 1-t
 with an AWS IAM User, which will be adopted and managed by Vault, including rotating it according
 to the configured `rotation_period`.
 
+In addition, this endpoint (Requires Vault Enterprise 1.19+) now supports cross account management.
```

Comment:
```suggestion
In addition, this endpoint supports cross account management.
```

I believe the recommendation from the Education team is to omit references like this since we have versioned docs.